README.md

```diff
@@ -1,11 +1,203 @@
-# Peggy
+# ETH Bridge Zone
 
-Peggy is a peg zone implementation that allows the movement of tokens and 
-potentially other assets from Ethereum (or any EVM) to Cosmos (or any
-Tendermint) chain. 
+[![CircleCI](https://circleci.com/gh/swishlabsco/cosmos-ethereum-bridge/tree/master.svg?style=svg)](https://circleci.com/gh/swishlabsco/cosmos-ethereum-bridge/tree/master)
```

Comment:
note: next push will replace this with cosmos/peggy. can I or denali get access to add to the cosmos circleCI, or can someone who has this take on the effort to set it up once merged?